go.mod

```diff
@@ -3,7 +3,13 @@ module github.com/decred/decrediton
 go 1.16
 
 require (
-	decred.org/dcrdex v0.0.0-20211128231427-0fc475d99afd
+	decred.org/dcrdex v0.0.0-20211217170953-795769123971
+	github.com/decred/dcrd/blockchain/standalone/v2 v2.0.1-0.20210914212651-723d86274b0d // indirect
+	github.com/decred/dcrd/certgen v1.1.2-0.20210914212651-723d86274b0d // indirect
+	github.com/decred/dcrd/chaincfg/chainhash v1.0.4-0.20210914212651-723d86274b0d // indirect
+	github.com/decred/dcrd/crypto/blake256 v1.0.1-0.20210914212651-723d86274b0d // indirect
+	github.com/decred/dcrd/crypto/ripemd160 v1.0.2-0.20210914212651-723d86274b0d // indirect
+	github.com/decred/dcrd/dcrec v1.0.1-0.20210914212651-723d86274b0d // indirect
```

Comment:
These indirect ones are weird.  If I remove them and `go mod tidy`, they do not get added, and the go.sum is updated to reflect the tagged ones that `decred.org/dcrdex` actually requires.
If I do `go get -d decred.org/dcrdex@release-v0.4` instead of just editing the go.mod and doing `go mod tidy`, I get almost the same result that you got.
Anyway, I'll merge this one very last module update shortly and try again: https://github.com/decred/dcrdex/pull/1359